pkg/controller/directvolumemigrationprogress/directvolumemigrationprogress_controller.go

```diff
@@ -247,14 +247,19 @@ func (r *ReconcileDirectVolumeMigrationProgress) reportContainerStatus(pvProgres
 	}
 
 	if containerStatus == nil {
-		pvProgress.Status.SetCondition(migapi.Condition{
-			Type:     InvalidPod,
-			Status:   migapi.True,
-			Reason:   NotFound,
-			Category: migapi.Critical,
-			Message: fmt.Sprintf("The spec.podRef %s must reference a `Pod` with container name %s",
-				path.Join(podRef.Namespace, podRef.Name), containerName),
-		})
+		if pod.Status.Phase == kapi.PodFailed {
```

Comment:
What happens when Pod phase is Unknown?

Reply:
Not sure if that ever happens while `containerStatus` is nil, but if it does then the controller will raise critical condition.